main.tf

```diff
@@ -14,10 +14,22 @@ variable "target_repository" {
   description = "The docker repo into which the image and attestations should be published."
 }
 
+variable "extra_repositories" {
+  default = []
+}
+
+variable "extra_keyring" {
```

Comment:
It kills me that this one is singular and the others are plural.

Reply:
I am hoping we can kill it all before too long, but you have to make it stupid fast first :)